ee/clickhouse/sql/plugin_log_entries.py

```diff
@@ -55,14 +58,14 @@
 instance_id,
 _timestamp,
 _offset
-FROM kafka_{table_name}
+FROM {database}.kafka_{table_name}
```

Comment:
most of our queries don't specify the database explicitly because they don't need to - we connect to the right one. Why here?

Reply:
I think this is explained in the comment at https://github.com/PostHog/posthog/blob/migrate/ee/clickhouse/sql/events.py#L58-L59